Packs/ProofpointServerProtection/README.md

```diff
@@ -0,0 +1,43 @@
+
+## Proofpoint Protection Server (PPS)
+
+This pack provides a Cortex XSOAR integration for Proofpoint Protection Server.
+Additionally, it supports Syslog-based log ingestion from Proofpoint Protection Server and includes parsing and modeling rules (XDM mapping) for Cortex XSIAM.
+
+## Configuration on Proofpoint Server Side
+1. Log in to the Proofpoint Protection Server interface.
+2. Click `Logs and Reports`.
+3. Click `Log Settings`.
+4. Go to the **Remote Log Options** panel.
+5. From `Syslog Protocol` Select **TCP** or **UDP**. 
```

Comment:
```suggestion
5. From `Syslog Protocol`, select **TCP** or **UDP**. 
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.